iroh-net/src/derp/codec.rs

```diff
@@ -677,6 +682,7 @@ mod tests {
     }
 }
 
+/// these test are slow in debug mode, so only run them in release mode
 #[cfg(test)]
 #[cfg(not(debug_assertions))]
```

Comment:
don't we compile with debug in release mode currently?

Reply:
I think we compile with debug symbols, but it seems enabling debug symbols does not set debug_assertions to true...

In any case we should do the ignore thing instead in a subsequent PR.

Reply:
❯ cargo test --release test_basic_read_write

running 1 test
test derp::codec::tests::test_basic_read_write ... ok

Just to confirm: the test is actually being run when run with --release...